format/spec.md

```diff
@@ -948,6 +961,7 @@ Lists must use the [3-level representation](https://github.com/apache/parquet-fo
 Notes:
 
 1. ORC's [TimestampColumnVector](https://orc.apache.org/api/hive-storage-api/org/apache/hadoop/hive/ql/exec/vector/TimestampColumnVector.html) consists of a time field (milliseconds since epoch) and a nanos field (nanoseconds within the second). Hence the milliseconds within the second are reported twice; once in the time field and again in the nanos field. The read adapter should only use milliseconds within the second from one of these fields. The write adapter should also report milliseconds within the second twice; once in the time field and again in the nanos field. ORC writer is expected to correctly consider millis information from one of the fields. More details at https://issues.apache.org/jira/browse/ORC-546
+2. ORC `timestamp` and `timestamp_instant` values store nanosecond precision. Iceberg ORC writers for Iceberg types `timestamp` and `timestamptz` truncate nanoseconds to microseconds.
```

Comment:
I think that this should state that Iceberg ORC writers MUST truncate nanoseconds to microseconds. Otherwise someone may store nanosecond values, which would break equality and have implications when casting from timestamp to timestamp_ns.

Reply:
Done. PTAL https://github.com/apache/iceberg/pull/8683/commits/150b6a9e19a45c0741c3d1cd563e92708983bf2d